src/handlers/transactions.ts

```diff
@@ -0,0 +1,150 @@
+import { Contract } from '@ethersproject/contracts';
+import axios from 'axios';
+import {
+  getLocalTransactionSignatures,
+  saveLocalTransactionSignatures,
+} from './localstorage/accountLocal';
+import { web3Provider } from './web3';
+import { ZerionTransaction } from '@rainbow-me/entities';
+
+interface FourByteResult {
+  bytes_signature?: string;
+  created_at?: string;
+  hex_signature?: string;
+  id: number;
+  text_signature?: string;
+}
+
+const REGISTRY_ADDRESS = '0x44691B39d1a75dC4E0A0346CBB15E310e6ED1E86';
+const abi = [
+  {
+    constant: false,
+    inputs: [{ name: '_new', type: 'address' }],
+    name: 'setOwner',
+    outputs: [],
+    payable: false,
+    type: 'function',
+  },
+  {
+    constant: true,
+    inputs: [],
+    name: 'totalSignatures',
+    outputs: [{ name: '', type: 'uint256' }],
+    payable: false,
+    type: 'function',
+  },
+  {
+    constant: true,
+    inputs: [],
+    name: 'owner',
+    outputs: [{ name: '', type: 'address' }],
+    payable: false,
+    type: 'function',
+  },
+  {
+    constant: false,
+    inputs: [],
+    name: 'drain',
+    outputs: [],
+    payable: false,
+    type: 'function',
+  },
+  {
+    constant: true,
+    inputs: [{ name: '', type: 'bytes4' }],
+    name: 'entries',
+    outputs: [{ name: '', type: 'string' }],
+    payable: false,
+    type: 'function',
+  },
+  {
+    constant: false,
+    inputs: [{ name: '_method', type: 'string' }],
+    name: 'register',
+    outputs: [{ name: '', type: 'bool' }],
+    payable: false,
+    type: 'function',
+  },
+  { inputs: [], type: 'constructor' },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, name: 'creator', type: 'address' },
+      { indexed: true, name: 'signature', type: 'bytes4' },
+      { indexed: false, name: 'method', type: 'string' },
+    ],
+    name: 'Registered',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, name: 'old', type: 'address' },
+      { indexed: true, name: 'current', type: 'address' },
+    ],
+    name: 'NewOwner',
+    type: 'event',
+  },
+];
+const parse = (signature: string) => {
```

Comment:
can we rename `parse` to something more descriptive - `parseSignatureToTitle` or something better